src/components/RulesTextBox/RulesTextBox.module.css

```
@@ -0,0 +1,18 @@
+.box {
+  height: fit-content;
+  width: 23rem;
+  border-radius: 28px;
+  background-color: rgba(0, 0, 0, 0.7);
+  color: var(--common-white);
+  border-color: var(--common-dark);
+  border-style: solid;

```

```suggestion
  border: 3px solid var(--common-white);
```
Obecnie nie będzie żadnej ramki, bo domyślna grubość to 0px.